cli/src/cmd/forge/test.rs

```diff
@@ -210,7 +242,7 @@ pub struct TestArgs {
     /// If the fuzz test does not fail, it will open the debugger on the last fuzz case.
     ///
     /// For more fine-grained control of which fuzz case is run, see forge run.
-    #[clap(long, value_name = "TEST FUNCTION")]
+    #[clap(long, value_name = "REGEX", help = "Run a test in the debugger")]
```

Comment:
the value name describes "the type of input the user should be using"
while REGEX would describe the type of the value, the value is expected to only match a single function, so I feel like REGEX would not be fitting here, TEST FUNCTION seems to be more accurate in this context IMO 

Also, this help message will make the doc string help message obsolete, which I think contains necessary information about this value.